core/kazoo_modb/src/kazoo_modb_migrate_maintenance.erl

```diff
@@ -0,0 +1,368 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2017, 2600Hz INC
+%%% @doc
+%%% The Great Kazoo Migration (TM)
+%%% @end
+%%%-------------------------------------------------------------------
+-module(kazoo_modb_migrate_maintenance).
+
+-export([migrate/0
+        ,migrate_account/1
+
+        ,migrate_voicemails/1
+        ,migrate_cdrs/1
+        ]).
+
+-include("kazoo_modb.hrl").
+
+-type update_funs() :: [fun((kz_json:object()) -> kz_json:object())].
+
+-spec migrate() -> 'ok'.
+migrate() ->
+    AccountIds = kz_term:shuffle_list(kapps_util:get_all_accounts('raw')),
+    Total = length(AccountIds),
+    io:format("Start migrating items to MODB from ~b databases~n~n", [Total]),
+    lists:foldl(fun(A, C) -> migrate_account_fold(A, C, Total) end, 1, AccountIds),
+    'ok'.
+
+-spec migrate_account_fold(ne_binary(), non_neg_integer(), non_neg_integer()) -> non_neg_integer().
+migrate_account_fold(AccountId, Current, Total) ->
+    io:format("(~p/~p) migrating items within account '~s'~n", [Current, Total, AccountId]),
+    _ = migrate_account(AccountId),
+    io:format("[~s] finished migrating items~n~n", [AccountId]),
+    Current + 1.
+
+-spec migrate_account(ne_binary()) -> 'ok'.
+migrate_account(Account) ->
+    lists:foreach(fun(Fun) -> Fun(Account) end
+                 ,[fun migrate_voicemails/1
+                  ,fun migrate_cdrs/1
+                  ]
+                 ).
+
+%%%===================================================================
+%%% Internal functions
+%%%===================================================================
+
+-spec get_account_id(ne_binary()) -> ne_binary().
```

Comment:
This looks like it's replicating `format_account_id(raw, _)`. Is this an accident?